CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
-surrounding the charge or take 70 explosive damage.
-Wall thicknesses:
-Wood - Up to 12in.
-concrete - up to 10in
-Reinforced Concrete - up to 8 in
-Steel plate - up to 2 in
+Wall Charge ($120):
+    Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
+surrounding the charge or take 70 explosive damage. Thicknesses of several 
+materials one wall charge can breach through are listed below.
 
-Det Cord ($230 per 2m of cord):
-Used to remove doors from hinges, act as ignition sources, enhance another 
-explosive, etc. Use 2m per door to be removed. .5m to wrap around a door 
-handle. 1m to wrap around a grenade. 4m to create a wall breaching pad (
-Explosives 50 DC)
+Wood:       Up to 12in.
+Concrete:   up to 10in
+Reinforced Concrete: up to 8 in
+Steel plate: up to 2 in
 
+Det Cord ($230 per 2m of cord):
+    Used to remove doors from hinges, act as ignition sources, enhance another 
+explosive, etc. Use 2 meters per door to be removed. One half meter to wrap 
+around a door handle. 1 meter to wrap around a grenade. 4 meters to create a 
+wall breaching pad (Explosives 50 DC).
 
+==============================
+Miscellaneous Items
+==============================
 
 Misc. Items and RP trinkets:
 
-Examples of RP trinkets include cigarettes, a lighter, cards, a wallet, an ID 
-pass, etc. Basically items that cannot be used in combat, but would have great 
-value in RP. These cost nothing, but you can only have a maximum of 2 items at 
-start.
+    Examples of RP trinkets include cigarettes, a lighter, cards, a wallet, an 
+ID pass, etc. Basically items that cannot be used in combat, but would have 
+great value in RP. These cost nothing, but you can only have a maximum of 2 
+items at start.
 
-==============================
-Miscellaneous Items
-==============================
+    Other misc. items are available for sale in the world. While you're playing 
+be on the lookout for shops or characters who might sell you more things. If you
+want to buy other starting equipment, ask your DM if something specific is 
+available and how much it might cost. Equipment that you start with generally
+depends on your character's fist mission. 
+
+20m of Nylon Rope ($30):
+    A light, sturdy rope 20 meters long which can hold a tension of 400lbs 
+without snapping.
 
-20m of Nylon Rope		$30:
-A light, sturdy rope 20 meters long which can hold a tension of 400lbs without snapping.
+Trulight LED flashlight	($22):
+    A metal black flashlight with bright LED bulbs. Illuminates up to 15m of 
+bright light.
 
-Trulight LED flashlight	$22:
-A metal black flashlight with bright LED bulbs. Illuminates up to 15m of bright light.
+Smart Phone	($180):
+    A touch screen, microphone and speaker in a plastic case. Reception may vary 
+by location.
 
-Smart Phone		$180:
-A touch screen, microphone and speaker in a plastic case. Reception may vary by location.
 DC to hack with skill points in hacking: 30
 DC to hack without skill points: 80
```

Comment:
Blocker: I am not certain what this will be read as. If we delete the preceding blank line will the parser interpret the DC lines as part of the preceding paragraph? (that would be good).

Reply:
I don't think you really want the DC's to be in the paragraph, do you? It would look like this.

        A touch screen, microphone and speaker in a plastic case. Reception may vary by location. DC to hack with skill points in hacking: 30 DC to hack without skill points: 80

Reply:
Disregard this one too.